presto-orc/src/main/java/com/facebook/presto/orc/reader/StreamReader.java

```diff
@@ -90,6 +94,7 @@ default Filter getFilter()
         return null;
     }
 
+    // TODO Remove this method
```

Comment:
Can we do it now?

Reply:
It is used in com.facebook.presto.orc.ColumnGroupReader#toString, hence, I need to change or remove that method. I'd like to do this separately.